src/main/java/org/folio/dcb/service/impl/BorrowingLibraryServiceImpl.java

```diff
@@ -89,6 +91,9 @@ public void updateStatusByTransactionEntity(TransactionEntity transactionEntity)
         log.info("updateStatusByTransactionEntity:: Item status is {} . So status of transaction is not updated",
           circulationItemRequest.getStatus().getName());
       }
+    } else if (ITEM_CHECKED_IN == transactionEntity.getStatus()) {
```

Comment:
Hi @DmytroBykov1 , I think we need to update the status from item_checked_in to closed only when we receive update request. If that is the case, this is not the right place. you need to add your code to updateTransactionStatus method. Thanks

Reply:
discussed